docs/sources/dashboards/variables/add-template-variables/index.md

```diff
@@ -97,6 +97,7 @@ Query expressions are different for each data source. For more information, refe
    - **On Time Range Change:** Queries the data source when the dashboard time range changes. Only use this option if your variable options query contains a time range filter or is dependent on the dashboard time range.
 1. In the **Query** field, enter a query.
    - The query field varies according to your data source. Some data sources have custom query editors.
+   - Make sure that the query returns values named `__text` and `__value` as appropriate in your query syntax. As a SQL-specific example, you can use a query such as `SELECT hostname AS __text, id AS __value FROM MyTable`; other query languages will vary depending on syntax.
```

Comment:
I've taken out the semi-colon and just made a new sentence at the end. But that last sentence doesn't entirely make sense to me. Does my revision work?
```suggestion
   - Make sure that the query returns values named `__text` and `__value` as appropriate in your query syntax. For example, in SQL, you can use a query such as `SELECT hostname AS __text, id AS __value FROM MyTable`. Queries for other languages will vary depending on syntax.
```